src/com/sphoorthi/LibraryManagement/Student.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package LibraryManagement;
+
+import java.util.*;
+
+/**
+ *
+ * @author Sphoorthi Gaddam
+ */
+public class Student {
+    
+    Map<String, String> students = new HashMap<String, String>();
+    Book book = new Book();
```

Comment:
This might not be required here since each time a student object is created, it will call book constructor and would create redundant / duplicate books.